test/adjoints.jl

```diff
@@ -72,3 +76,4 @@ loss(x)
 @test Zygote.gradient(loss6, x)[1] == ForwardDiff.gradient(loss6, x)
 @test Zygote.gradient(loss7, x)[1] == ForwardDiff.gradient(loss7, x)
 @test Zygote.gradient(loss8, x)[1] == ForwardDiff.gradient(loss8, x)
+@test ForwardDiff.derivative(loss9, 0.0) == VectorOfArray([collect(3i:3i+3) for i in 1:5])
```

Comment:
if you pass in zero, isn't the derivative zero? Is this meant to be 1?

Reply:
It's a linear function right? `f(x) = 3x` has a derivative of 3

Reply:
😅